inductiva/molecules/scenarios/protein_solvation/protein_solvation.py

```diff
@@ -189,6 +197,14 @@ def compute_charge(self,
         is_charged = abs(charge) > 1e-6
         return is_charged
 
+    def download_outputs(self, output_dir: str = None, resource_pool_id: Optional[UUID] = None):
```

Comment:
This will only work when you do the simulate_async method so maybe say that in the docstring

Reply:
Maybe this can go to the base scenario, no ? WDYT?